src/components/TalkDetails.tsx

```diff
@@ -1,11 +1,19 @@
 import { gql, useQuery } from "@apollo/client";
 import React from "react";
+import { VSpace } from "./layout";
+import { AgendaTalksListItemSpeakers } from "./agenda/AgendaTalksListItem";
+import { css } from "emotion";
 
 export const TalkDetailsFragment = gql`
   fragment TalkDetails on Talk {
     id
     title
+    abstract
     description
+    speakers {
+      id
```

Comment:
Nit: This is fine for now, but typically when using GraphQL, if you're loading data for a component, that component will define a fragment which is then re-usable (and not brittle).

Reply:
I replaced it with the AgendaListItemQuery, but not sure if that's the correct approach either

Reply:
Hmm yeah I think my feedback was a bit ambiguous. Sorry about that. :^) It was better before, and I really just had one nitpick:

You had this (IIRC):
```graphql
query TalkDetailsQuery {
  ...
  # These three fields are selected because they're what's needed by the speakers component
  speakers { id, name, avatar }
}
```

and the _better_ thing to do is this:
```graphql
query TalkDetailsQuery {
  ...
  # SpeakerInfo is a fragment that's already defined
  speakers { ...SpeakerInfo }
}
```

This means that if the `SpeakerInfo` fragment changes in the future (e.g., to request their GitHub name to link to as well), we only have to update the one place. Typically there's a 1-to-1 relationship between data-fetching components and GraphQL fragments.

Reply:
I moved the query into a file similar to the other queries and now the dependency on `SpeakerInfo` seems to be working whereas it wouldn't purely by reference in the inline query.  Should a `talk` folder be created? `TalkDetails.tsx` was a pre-existing file in that location so I built it out there, but now that there are a few files, a folder might be better.